packages/flutter/lib/src/rendering/binding.dart

```diff
@@ -10,6 +10,7 @@ import 'package:flutter/scheduler.dart';
 import 'package:flutter/semantics.dart';
 import 'package:flutter/services.dart';
 
+import '../widgets/drag_target.dart';
```

Comment:
This is a layering violation. You can't import anything from widgets into the lower-level rendering library.

Reply:
I am aware of this, but not sure how to circumvent this.

I can't put the `DragTargetPostFrameUpdater` in another file, since it requires access to the private fields of `_DragTarget`.

Alternatively i can think of
- changing `_DragTarget`s visibility, and moving `DragTargetPostFrameUpdater`
- registering normal post frame callbacks within `drag_target.dart` instead of using a `binding.dart` persistent callback